types/validation.go

```diff
@@ -57,8 +57,14 @@ func VerifyCommit(chainID string, vals *ValidatorSet, blockID BlockID,
 //
 // This method is primarily used by the light client and does not check all the
 // signatures.
```

Comment:
Remove "does not check all the signatures." because this now depends on `countAllSignatures`. Similarly for `VerifyCommitLightTrusting` below.